### ISSUE TITLE
gtm-script, noscript appended injection issue

### ISSUE THREAD

**Issue by hagi4u** [nuxt-community/gtm-module#16]
hello there,
i found #gtm-script and noscript appended injection issue.

this issue is nuxt start when production mode and page refresh 
(or someone are viewing every single pages)

## gtm-script
![75516515-09614d80-5a40-11ea-8742-3616a6b05af5](https://user-images.githubusercontent.com/6129756/75526892-fa859580-5a55-11ea-97cb-0e4bb59fb180.png)

## noscript
![75516386-aff91e80-5a3f-11ea-8f90-2dcb18a0eae8](https://user-images.githubusercontent.com/6129756/75526907-ffe2e000-5a55-11ea-938e-19d7e709c1bb.png)

**Comment by hagi4u** [comment]
my `nuxt.config.js` 
```JavaScript
...
modules: [
    ...
    [
      '@nuxtjs/gtm',
      {
        autoInit: false,
      },
    ],
    ...
]
...
```

**Comment by hagi4u** [comment]
i found issue that cause codes and i think about code is `lib/plugins.js` 32-34 lines

```JavaScript
if (inits.length) {
  gtmScript.innerHTML += `;${JSON.stringify(inits)}.forEach(function(i){window._gtm_inject(i)})`
}
```

I guess `$gtm.init('GTM-XXXXXX');` code in `~/plugins/gtm.js` then every single route execute `$gtm.init()`. 
so `gtmScript.innerHTML += ..` was appended `window._gtm_inject` code when load page to every single route

**Comment by pi0** [comment]
Hi @hagi4u. Actually `_gtm_inject` prevents calling with same id multiple times: 

https://github.com/nuxt-community/gtm-module/blob/master/lib/module.js#L41

`if(w.doNotTrack||w[x][i])return;w[x][i]=1;${injectScript};}`

Also, plugins are called once on client/server not on each route change...

Would you please provide a way to reproduce this issue?

**Comment by hagi4u** [comment]
@pi0 okay so i prepared to [reproduce repo](https://github.com/hagi4u/reproduce-nuxtjs-gtm-issue)

you can do check that how to yarn build and yarn start

1. run browser and inspector
2. refresh pages
3. check `[data-hid="gtm-script"]` and `[data-hid="gtm-noscript"]` tags

![image](https://user-images.githubusercontent.com/6129756/75732602-470ef080-5d36-11ea-83cd-3158153a4bf8.png)

**Comment by pi0** [comment]
Thanks @hagi4u for repro. I think it's a serious issue already fixed (https://github.com/nuxt/nuxt.js/pull/6807) and will be released ASAP with nuxt 2.12.0.

For now, best is to use `nuxt-edge`
